### ISSUE TITLE
Can't convert string to float '1,006.73' 

### ISSUE THREAD

**Issue by martinffx** [pydata/pandas-datareader#345]
Amazon is too dam valuable!

```python
>>> import pandas_datareader.data as web
>>>
>>> q = web.get_quote_google(['AMZN', 'GOOG'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/martinrichards/code/play/stock-scraper/venv/lib/python3.6/site-packages/pandas_datareader/data.py", line 56, in get_quote_google
    return GoogleQuotesReader(*args, **kwargs).read()
  File "/Users/martinrichards/code/play/stock-scraper/venv/lib/python3.6/site-packages/pandas_datareader/base.py", line 69, in read
    return self._read_one_data(self.url, self.params)
  File "/Users/martinrichards/code/play/stock-scraper/venv/lib/python3.6/site-packages/pandas_datareader/base.py", line 79, in _read_one_data
    return self._read_lines(out)
  File "/Users/martinrichards/code/play/stock-scraper/venv/lib/python3.6/site-packages/pandas_datareader/google/quotes.py", line 33, in _read_lines
    for x in result], columns=['change_pct',
  File "/Users/martinrichards/code/play/stock-scraper/venv/lib/python3.6/site-packages/pandas_datareader/google/quotes.py", line 33, in <listcomp>
    for x in result], columns=['change_pct',
ValueError: could not convert string to float: '1,006.73'
```

Should be a simple fix, I'll try my hand at it and submit a PR.

**Comment by martinffx** [comment]
Lots of test failures:

```
Ran 80 tests in 158.580s

FAILED (errors=17, failures=2)
```

**Comment by femtotrader** [comment]
This issue should be solved by https://github.com/pydata/pandas-datareader/pull/335

**Comment by martinffx** [comment]
ahh, was about to submit that exact fix as a PR. Any reason it hasn't been merged? Has the author not update with the PEP8 comment?

**Comment by femtotrader** [comment]
I can't merge PRs that don't pass all unit tests
and I'm quite busy this year to fix all tests 

Pinging @jreback @davidastephens @gfyoung

**Comment by jreback** [comment]
closed by #335
